amplipi/updater/asgi.py

```diff
@@ -95,6 +113,90 @@ class ReleaseInfo(BaseModel):
   logger.exception(f'Error loading identity file: {e}')
 
 
+class Persist_Logs(BaseModel):
+  """Basemodel that consists of a bool and int, used to change different config files around the system via POST /settings/persist_logs"""
+  persist_logs: bool
+  auto_off_delay: int
+
+
+@router.get("/settings/persist_logs")
+def get_log_persist_state():
+  """
+  Checks /etc/systemd/journald.conf to find if the current storage setting is persistent and returns a bool
+  Note that returning false doesn't necessarily mean that logs are set to volatile, and could just mean that the config file is missing the line being read
+  """
+  create_logging_ini()
```

Comment:
does this `GET` not set the `auto_off_delay` to 14 every time?

Reply:
No, that's just a fallback value. A fallback value is only used if the section or header being read can't be found, in this case it will yield 14 and then any future `POST` requests will create both header and section so it will be properly read in the future

Basically it protects against edge cases where the file was deleted somehow, or the deploy script had an error.